apps/web-client/src/routes/outbound/+page.svelte

```diff
@@ -33,8 +33,9 @@
 
 	let initialized = false;
 	onMount(() => {
-		if (status) firstValueFrom(outNoteListStream).then(() => (initialized = true));
-		else {
+		if (status) {
+			firstValueFrom(outNoteListStream).then(() => (initialized = true));
```

Comment:
Ok, I'd like to go back on my suggestion for using the variant `{ ready: false, notes: [] } | { ready: true, notes: NavEntry[]` (or whichever the type for note list), as I see now that `firstValueFrom` is used on a stream, not on the `readable` from said stream - so, yeah, it makes sense to address that at the DB interface level.

My suggestion would still work, I guess, but it also make sense to untangle the streams a bit (if that's what you're going with)